src/mono/browser/runtime/dotnet.d.ts

```diff
@@ -336,7 +336,7 @@ interface AssetEntry {
      */
     pendingDownload?: LoadingResource;
 }
-type SingleAssetBehaviors = 
+type SingleAssetBehaviors =
```

Comment:
These types need to be edited in `src\mono\browser\runtime\types\index.ts` and the `dotnet.d.ts` generated rather than edited by hand. Use `npm run rollup` or build in Debug mode in order to trigger it.